### DISCUSSION TITLE
FileZilla

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#13] [Requests] [answered]
FileZilla website: https://filezilla-project.org/

FileZilla privacy policy: https://filezilla-project.org/privacy.php

Logo: https://upload.wikimedia.org/wikipedia/commons/0/01/FileZilla_logo.svg

Description: "FileZilla is a free and open-source, cross-platform FTP application, consisting of FileZilla Client and FileZilla Server. Clients are available for Windows, Linux, and macOS, servers are available for Windows only. Both server and client support FTP and FTPS (FTP over SSL/TLS), while the client can in addition connect to SFTP servers.

FileZilla's source code is hosted on SourceForge and the project was featured as Project of the Month in November 2003. However, there have been criticisms that SourceForge bundles malicious software with the application."

**Comment by mayrinck** [comment, marked as answer]
> FileZilla's source code is hosted on SourceForge and the project was featured as Project of the Month in November 2003. However, there have been criticisms that SourceForge bundles malicious software with the application.

I think this is the main reason why we should avoid adding filezilla for now, but it's a good suggestion. As soon as I manage to add other apps, I will get to this one...
